pyramid/session.py

```diff
@@ -187,6 +188,9 @@ def BaseCookieSessionFactory(
       Hide the cookie from Javascript by setting the 'HttpOnly' flag of the
       session cookie. Default: ``False``.
 
+    ``samesite``
+      The 'samesite' option of the session cookie. Default ``'Lax'``.
```

Comment:
Could we enumerate here the other valid options for samesite or is that not worthwhile? Is `None` valid to make a cookie without the option set? It would be worth mentioning one way or the other in the docs. Also as @stevepiercy mentioned on the old issue a `.. versionchanged: 1.10` should be in the docstrings of the public apis.

Reply:
Setting the `SameSite` to `None` will not set it in the cookie.

Reply:
Yes, we could enumerate the other valid options here if we want.